systests/spring-boot/src/test/java/org/apache/cxf/systest/jaxws/spring/boot/SpringJaxwsTimedTest.java

```diff
@@ -0,0 +1,271 @@
+
+    @EnableAutoConfiguration
+    static class TestConfig {
+        @Autowired
+        private Bus bus;
+
+        @Autowired
+        private MetricsProvider metricsProvider;
+
+        @Bean
+        public Endpoint helloEndpoint() {
+            EndpointImpl endpoint = new EndpointImpl(bus, new HelloServiceTimedImpl(), null, null, new MetricsFeature[]{
+                new MetricsFeature(metricsProvider)
+            });
+            endpoint.publish("/" + HELLO_SERVICE_NAME_V1);
+            return endpoint;
+        }
+        
+        @Bean
+        public Endpoint secondHelloEndpoint() {
+            EndpointImpl endpoint = new EndpointImpl(bus, new HelloServiceEmptyTimedImpl(), null, null, 
+                new MetricsFeature[] {
+                    new MetricsFeature(metricsProvider)
+                }
+            );
+            endpoint.publish("/" + HELLO_SERVICE_NAME_V2);
+            return endpoint;
+        }
+
+    }
+
+    @AfterEach
+    public void clear() {
+        registry.clear();
+    }
+
+    @Test
+    public void testJaxwsTimedSuccessMetric() throws MalformedURLException {
+        // given in setUp
+
+        // when
+        String actual = sendSoapRequest(DUMMY_REQUEST_BODY, HELLO_SERVICE_NAME_V1);
+
+        // then
+        assertThat(actual)
+                .isEqualTo("<ns2:sayHelloResponse xmlns:ns2=\"http://service.ws.sample/\">"
+                        + "<return>Hello, Elan</return>"
+                        + "</ns2:sayHelloResponse>");
+
+        await()
+            .atMost(Duration.ofSeconds(1))
+            .ignoreException(MeterNotFoundException.class)
+            .until(() -> registry.get("cxf.server.requests").timers(), not(empty()));
+        RequiredSearch serverRequestMetrics = registry.get("cxf.server.requests");
+
+        Map<Object, Object> serverTags = serverRequestMetrics.timer().getId().getTags().stream()
+                .collect(toMap(Tag::getKey, Tag::getValue));
+
+        assertThat(serverTags)
+            .containsOnly(
+                entry("exception", "None"),
+                entry("faultCode", "None"),
+                entry("method", "POST"),
+                entry("operation", "sayHello"),
+                entry("uri", "/Service/" + HELLO_SERVICE_NAME_V1),
+                entry("outcome", "SUCCESS"),
+                entry("status", "200"));
+        
+        RequiredSearch timedMetrics = registry.get("sayHello.requests");
+
+        Map<Object, Object> timedTags = timedMetrics.timer().getId().getTags().stream()
+                .collect(toMap(Tag::getKey, Tag::getValue));
+
+        assertThat(timedTags)
+            .containsOnly(
+                entry("exception", "None"),
+                entry("faultCode", "None"),
+                entry("method", "POST"),
+                entry("operation", "sayHello"),
+                entry("uri", "/Service/" + HELLO_SERVICE_NAME_V1),
+                entry("outcome", "SUCCESS"),
+                entry("status", "200"));
```

Comment:
Thanks for the fix! 

General question, I'm not sure myself what would be best / most expexted: Should `sayHello.requests` contain all those "cxf related" tags or just the tags provided by the user e.g. with `@Timed(extraTags="...")`?